bentoctl/terraform.py

```diff
@@ -0,0 +1,63 @@
+import os
+import json
+import subprocess
+
+from bentoctl.exceptions import BentoctlException
+
+TERRAFORM_VALUES_FILE = "bentoctl.tfvars"
+TERRAFORM_INIT_FOLDER = ".terraform"
+
+
+def terraform_run(cmd: list, return_output: bool = False):
+    try:
+        if not return_output:
+            subprocess.run(["terraform", *cmd], check=False)
+        proc = subprocess.Popen(
+            ["terraform", *cmd],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        stdout, stderr = proc.communicate()
+        return proc.returncode, stdout.decode("utf-8"), stderr.decode("utf-8")
```

Comment:
should we pipe out the outputs and stderr from terraform? we want the users to know exactly what terraform is doing write?

Reply:
Doesn't user already see the changes in their console with subprocess?

Reply:
ohh the issue was there were two subprocess calls happening if return_output if false.